src/NSwag.Generation.WebApi/Processors/OperationParameterProcessor.cs

```diff
@@ -495,14 +495,6 @@ private OpenApiParameter AddBodyParameter(OperationProcessorContext context, str
                 {
                     operationParameter.Default = defaultValue;
                 }
-                else if (operationParameter.Schema.HasReference)
```

Comment:
I think we need to keep this in case there is actually a default value, but only then: 

    else if (operationParameter.Schema.HasReference && defaultValue != null)